test/benchmarks/material/button/app.module.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {A11yModule} from '@angular/cdk/a11y';
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {MatButtonModule} from '@angular/material/button';
+
+/**
+ * @title Button benchmark component.
+ */
```

Comment:
While addressing other feedback, you could remove the `@title` comment here (as it's a leftover from doc examples)